common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -99,3 +99,18 @@
 else:
     tmpl.render_context(context)
 %></%def>
+
+<%def name="optional_include_mako_with_fallback(file, fallback, with_microsite=False, **kwargs)"><%
```

Comment:
@dpowell string_or_list argument could be a solution but for me it looks like a little bit unpythonic. The philosophy of python is based on the couple of principles. "Explicit is better than implicit" one of them. So as for my opinion if function always expects an argument as a list it is more explicit (even this list consists of only one element).